CHANGELOG.adoc

```diff
@@ -42,6 +42,11 @@ A reset of the database is needed.
 - https://github.com/eclipse-syson/syson/issues/356[#356] [action-flow-view] The Allocate edge is not displayed in the Action Flow View diagram.
 - https://github.com/eclipse-syson/syson/issues/403[#403] [diagrams] Wrong computation of the Succession owner
 - https://github.com/eclipse-syson/syson/issues/407[#407] [diagrams] Fix the owner membership of dropped elements
+- https://github.com/eclipse-syson/syson/issues/376[#376] [export] Membership names are not correctly deresolved during the export
```

Comment:
Can you change all the sentences here with something like : `Fix an issue where....` instead of just naming the problem, please?

Reply:
Done